com.unity.multiplayer.mlapi/Runtime/Profiling/ITransportProfilerData.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace MLAPI.Profiling
+{
+    public interface ITransportProfilerData
+    {
+        Dictionary<string, int> GetTransportGetData();
```

Comment:
I'm still not sold on returning a Dictionary<>, because this means either the following:

- Transport needs to allocate new memory for every network tick
- Transport reuses memory, but hands it off to 3rd party in a potentially non-readonly way

I would propose either of these adjustments:

- The interface _receives_ a dictionary in the interface, with the expectation that additional values are filled
- The interface returns an IReadOnlyDictionary, to enforce that the dictionary is not modified by the third party

Reply:
updated to `IReadOnlyDictionary `